src/SearchForm.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+
+export default function SearchForm({initialQuery,onSearch}) {
+
+    const [query,SetQuery] = useState(initialQuery);
+
+    const handleInputChange = (event) => {
+        SetQuery(event.target.value);
+      };
+
+      const handleSearch =() =>{
+        onSearch(query);
+      }
+
+      const handleKeyPress = (event) => {
+        if (event.key === 'Enter') {
+          onSearch(query);
+        }
+      };
+
+    return (
+        <div className="container my-5">
```

Comment:
Instead of having keyPress, you can wrap the form with <form> tag and add an onSubmit handler. You can avoid the click handler on button as well

Reply:
Resolved